transcom/mymove
# [MB-16355] adding origin and destination fsc

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16355)

## Summary

This PR adds new labels and values to display calculations for DDSFSC, DOSFSC, and FRA in the TIO document viewer

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the `PARAMS` move as the prime
2. Request payment for `DOSFSC` & `DDSFSC`
3. Access the same move as a `TIO`
4.  You'll have to hit the `Review Weights` button and bump the `Max billable weight` up to 14k so it allow you to review service item. You can click out with the x in the top left corner once saved.
5. Scroll to the bottom and click the `review service items` button for the `DOSFSC` & `DDSFSC` surcharge
6. Hit the `show calculations` button for the `DOSFSC` button and you should see the expected calculations displayed.
7. Hit the next button below after verifying and repeat step 6.

### Frontend


## Screenshots
DDSFSC:
![image](https://github.com/transcom/mymove/assets/111928238/98f8cf6f-4971-417a-8d62-6e9cb34f28e3)

DOSFSC:
![image](https://github.com/transcom/mymove/assets/111928238/ce54239d-7352-420b-9ec9-582bee33c39b)
